script.js

```diff
@@ -0,0 +1,59 @@
+const form = document.querySelector('#form');
+const bookList = document.querySelector('.book-list');
+const titleInput = document.querySelector('#input_title');
+const authorInput = document.querySelector('#input_author');
+const addBtn = document.querySelector('#add_btn');
+
+let collections = JSON.parse(localStorage.getItem('collections')) || [];
+
+function addBook() {
+  collections.push({
+    title: titleInput.value,
+    author: authorInput.value,
+  });
+  localStorage.setItem('collections', JSON.stringify(collections));
+}
+
+function showBook() {
+  bookList.innerHTML = '';
+  collections.forEach((collection, index) => {
+    bookList.innerHTML += `<div class='book${index + 1}' id="${index}">
+            <p class="book-info">
+                <span class="title">${collection.title}</span>
+                <br/>
+                <span class="author">${collection.author}</span>
+            </p>
+            <button class="remove-btn">Remove</button>
+            <hr/>
+    </div>`;
+  });
+}
+
+function removeBook(cur) {
```

Comment:
Great job with the project. However, this part of the application is breaking. I have recorded [this video](https://www.loom.com/share/308e5d43f90e40b89fd456bfacdc8ce5) where I demonstrated what exactly is the issue. Kindly fix this error and then submit it for a re-review.